int32 index # index to remove
---
bool success # indicate successful run of triggered service
string message # informational, e.g. for error messages